doc/dev/osd_internals/backfill_reservation.rst

```diff
@@ -39,7 +39,7 @@ which PGs are recovered.  Admins can override the default order by using
 ``force-recovery`` or ``force-backfill``. A ``force-recovery`` with op
 priority ``255`` will start before a ``force-backfill`` op at priority ``254``.
 
-If a recovery is needed because a PG is below ``min_size`` a base priority of
+If a recovery op is needed because a PG is below ``min_size`` a base priority of
```

Comment:
"If recovery is needed" makes more sense.

Reply:
Agreed.